wagtail_ab_testing/test/tests/test_migrations.py

```diff
@@ -0,0 +1,19 @@
+from io import StringIO
+from django.core import management
+from django.test import TestCase
+
+
+class TestReportView(TestCase):
```

Comment:
I believe the test class name here needs to be updated...

Reply:
Whoops, copypasta fail :) Thanks for spotting this!